src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/ValidationMetadata.cs

```diff
@@ -19,6 +19,14 @@ public class ValidationMetadata
         public bool? IsRequired { get; set; }
 
         /// <summary>
+        /// Gets or sets a value that indicates whether children of the model should be validated. If <c>null</c>
+        /// then <see cref="ModelMetadata.ValidateChildren"/> will be computed based on the value of
+        /// <see cref="ModelMetadata.IsComplexType"/> and <see cref="ModelMetadata.IsEnumerableType"/>. By default
+        /// complex types and enumerable types support validation of children.
+        /// </summary>
+        public bool? ValidateChildren { get; set; }
```

Comment:
Reads fairly well though the last sentence switches perspectives. Suggest: `... and <see cref="ModelMetadata.IsEnumerableType"/>: <see cref="ModelMetadata.ValidateChildren"/> will be <c>true</c> if either <see cref="ModelMetadata.IsComplexType"/> or <see cref="ModelMetadata.IsEnumerableType"/> is <c>true</c>; <c>false</c> otherwise.`

Hmm, that's a bit wordy. Follow your :heart: